static/src/javascripts/projects/common/modules/analytics/google.js

```diff
@@ -0,0 +1,139 @@
+// @flow
+
+import config from 'lib/config';
+import mediator from 'lib/mediator';
+
+const trackerName = config.googleAnalytics.trackers.editorial;
+const send = `${trackerName}.send`;
+
+const extractLinkText = (el: ?HTMLElement): ?string => {
```

Comment:
I mean, it does not do anything. Another instance where `Maybe<A>` would so much more make sense.